internal/provisioner/kops_provisioner_installation_sli.go

```diff
@@ -58,11 +60,19 @@ func (provisioner *KopsProvisioner) makeSLIs(clusterInstallation *model.ClusterI
 
 func (provisioner *KopsProvisioner) createInstallationSLI(clusterInstallation *model.ClusterInstallation, k8sClient *k8s.KubeClient) error {
 	wait := 60
+	logger := provisioner.logger.WithFields(log.Fields{
+		"cluster":      clusterInstallation.ClusterID,
+		"installation": clusterInstallation.InstallationID,
+	})
 	sli := provisioner.makeSLIs(clusterInstallation)
 	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(wait)*time.Second)
 	defer cancel()
 	_, err := k8sClient.SlothClientsetV1.SlothV1().PrometheusServiceLevels("prometheus").Create(ctx, sli, metav1.CreateOptions{})
-	if err != nil {
+	if err != nil && k8sErrors.IsNotFound(err) {
+		logger.Debugf("Sloth CRD doesn't exist on cluster: %s", err)
+		return nil
+	}
+	if err != nil && !k8sErrors.IsNotFound(err) {
```

Comment:
If `IsNotFound(...)` we would already return.
```suggestion
	if err != nil {
```